lib/create_api_gem/forms/settings/settings.rb

```diff
@@ -67,14 +69,19 @@ def payload
   def same?(actual)
     (redirect_after_submit_url.nil? || redirect_after_submit_url == actual.redirect_after_submit_url) &&
       (google_analytics.nil? || google_analytics == actual.google_analytics) &&
+      (facebook_pixel.nil? || facebook_pixel == actual.facebook_pixel) &&
       (notifications.nil? || notifications.same?(actual.notifications)) &&
       (description.nil? || description == actual.description) &&
-      (image.nil? || image[:href].start_with?("#{APIConfig.image_api_request_url}/images/") && actual.image[:href].start_with?("#{APIConfig.image_api_request_url}/images/")) &&
       (show_typeform_branding.nil? ? Settings.default.show_typeform_branding : show_typeform_branding) == actual.show_typeform_branding &&
       (progress_bar.nil? ? Settings.default.progress_bar : progress_bar) == actual.progress_bar &&
       (language.nil? ? Settings.default.language : language) == actual.language &&
       (is_public.nil? ? Settings.default.is_public : is_public) == actual.is_public &&
-      (allow_indexing.nil? ? Settings.default.allow_indexing : allow_indexing) == actual.allow_indexing
+      (allow_indexing.nil? ? Settings.default.allow_indexing : allow_indexing) == actual.allow_indexing &&
+      same_image?(actual)
```

Comment:
☝️ this is necessary because Rubocop was complaining there are too many conditionals in the `same?` method

Reply:
🔥